dev/src/main/java/org/eclipse/codewind/intellij/ui/CodewindToolWindow.java

```diff
@@ -223,4 +226,18 @@ private void handleApplicationPopup(CodewindApplication application, Component c
         ActionPopupMenu popupMenu = ActionManager.getInstance().createActionPopupMenu("CodewindTree", actions);
         popupMenu.getComponent().show(component, x, y);
     }
+
+    private void handleConnectionManagerPopup(ConnectionManager manager, Component component, int x, int y) {
+        DefaultActionGroup actions = new DefaultActionGroup("CodewindApplicationGroup", true);
+
+        InstallStatus status = manager.getLocalConnection().getInstallStatus();
+        if (status.isInstalled()) {
+            actions.add(updateCodewindAction);
```

Comment:
We should model the behaviour on the Eclipse plugin

Update shouldn't be an action on the menu if there is no update available.

![image](https://user-images.githubusercontent.com/29612372/73758437-99a9bc80-4738-11ea-9b3a-7fc4b8669922.png)

If  an update is needed, the status beside the 'Local' node changes to inform the user that they need to update (the intellij plugins already shows this status).

![image](https://user-images.githubusercontent.com/29612372/73758690-f5744580-4738-11ea-9103-e02061e33dbc.png)

and the popup then shows 'Update' instead of 'Uninstall'

![image](https://user-images.githubusercontent.com/29612372/73758863-294f6b00-4739-11ea-8506-d12eb3288477.png)

Reply:
Other than "... the status beside the 'Local' node changes ...", all of the other changes were implemented.